analysis/measures_definition_pf_codes_conditions.py

```diff
@@ -66,14 +66,16 @@
 # Age bands for age breakdown
 age = patients.age_on(INTERVAL.start_date)
 age_band = case(
+    when(age.is_null()).then("Missing"),
     when((age >= 0) & (age < 20)).then("0-19"),
     when((age >= 20) & (age < 40)).then("20-39"),
     when((age >= 40) & (age < 60)).then("40-59"),
     when((age >= 60) & (age < 80)).then("60-79"),
     when(age >= 80).then("80+"),
-    when(age.is_null()).then("Missing"),
+    otherwise="Missing"
 )
 
+
```

Comment:
looks like you went back to the original, if you went back to the original it would not pick it up as a change in the PR and make it a bit easier to review